fp.js

```diff
@@ -9,7 +9,7 @@ var people = [
 people.forEach(person => {
   peoplesAges.push(person.age)
 })
-
+var peoplesAges = people.map(peoplesAges);
```

Comment:
no quite, `.map` takes a function as an argument, and the first parameter to the function you pass it will represent *each* item in the array. i.e.
```js
var peoplesAges = people.map(function (person) {
  return person.age
})
```